### DISCUSSION TITLE
Question about translation

### DISCUSSION THREAD

**Discussion by luxdev01** [StellarSand/IYPS#27] [Q&A] [answered]
Hi, I'm translating the app into Brazilian Portuguese, I translated almost all the strings, except for these 2 strings:

```
<string name="turns">\u2022 Turns<string>
```
```
<string name="shifted_times">\u2022 Shifted %1$s times<string>
```

I couldn't find these strings in the app to know the right context to translate.

**Comment by StellarSand** [comment, marked as answer]
Hello, thanks for the translations. Both these strings would be shown in `Match sequence`.
- `qwertyuiopoiuytrewq` will show:
    - `2 turns`: This is because the sequence follows 2 turns across the top row of the keyboard — first from left to right (`qwertyuiop`), and then back from right to left (`oiuytrewq`).
    - `Shifted 0 times`: As shift key wasn't used.
- `!@#$%$#@!` will show:
    - `2 turns`: Similar to the first example, this password moves across the row of numbers — first from left to right (`!@#$%`) and then back from right to left (`$#@!`).
    - `Shifted 9 times`: As shift key was used to type the whole sequence.

**Reply by luxdev01** [reply]
Thanks for clarifying!

I will translate the string "Shifted %1$s times" to "Tecla Shift usada %1$s vezes"(that means "Shift key used %1$s times), as the word Shift does not have a translation.

**Reply by StellarSand** [reply]
Sure. :)
